AdamGrzybkowski/WordPress-Android
# Follow recommended blog from "recommended_blogs" discovery card

Initial draft PR to discuss the code 

Questions:
1. Should we show "enable notification" snack when following the Site from the "recommended_blogs" card?


To test:
- open Discovery tab
- scroll to 'Sites to follow' card
- click follow/unfollow IconButton
- verify site was followed/unfollowed

PR submission checklist:
